### DISCUSSION TITLE
getting error while running my modflow model

### DISCUSSION THREAD

**Discussion by zargol7419** [modflowpy/flopy#1550] [Q&A] [answered]
Hi,
I am using flopy package to execute my existing model using run_model code, but when it comes to running the model I've got this error, and I don't know what is the problem. Could you please help me in this regard? I really appreciate your answers.
Here is the executed code:
![image](https://user-images.githubusercontent.com/103249900/190902918-6e2879b6-71c0-496c-b1bf-a663a267f1ca.png)

**Comment by jlarsen-usgs** [comment, marked as answer]
The reason that it is crashing is that your call signiture is incorrect. From the `run_model` documentation:

```
exe_name : str
        Executable name (with path, if necessary) to run.
namefile : str
        Namefile of model to run. The namefile must be the
        filename of the namefile without the path. Namefile can be None
        to allow programs that do not require a control file (name file)
        to be passed as a command line argument.
model_ws : str
        Path to the location of the namefile. (default is the
        current working directory - './')
````

psuedocode example:

```python
success, buff = flopy.run_model(
    namefile="mynamefile.nam", 
    exe_name="path/to/my/modflow.exe", 
    model_ws="path/to/the/directory/that/contains/my/namefile"
)
```
